src/pages/Patients/Patients.js

```diff
@@ -569,28 +586,31 @@ const Patients = () => {
           </LinkButton>
         </div>
       </div>
-      {isShowPatientInfoColumn && (
-        <PatientInfoColumn className="bg-white">
-          <InfoValue className="m-0 px-3">
-            {patients?.length ?? 0} active patients
-          </InfoValue>
+
+      <PatientInfoColumn className="bg-white">
+        <InfoValue className="m-0 px-3">
+          {patients?.length ?? 0} active patients
```

Comment:
correct me if I am wrong, but shouldn't we also need to subtract with discharged patients here?

Reply:
resolved